packages/usdk/packages/upstreet-agent/packages/react-agents/lib/discord/discord-client.js

```diff
@@ -434,6 +461,8 @@ export class DiscordBotClient extends EventTarget {
         } = j;
         switch (method) {
           case 'ready': {
+            this.input.setUsername(this.name);
```

Comment:
Maybe support missing name/previewUrl checking to be safe?

Reply:
added check